packages/routing-utils/src/superstatic.ts

```diff
@@ -5,6 +5,11 @@
 import { parse as parseUrl, format as formatUrl } from 'url';
 import { pathToRegexp, compile, Key } from 'path-to-regexp';
 import { Route, Redirect, Rewrite, Header } from './types';
+import { HasField } from '.';
```

Comment:
HasField can be moved up to the `./types` import above

Reply:
Updated